### ISSUE TITLE
USER_AREA is defined true in menu manager - conflicts with e_header

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#4555]
## Bug Description
this is recommended solution for not inserting e_header css in admin area

`if(deftrue('USER_AREA')) // prevents inclusion of JS/CSS/meta in the admin area.`

but USER_AREA is defined true in preview (menu manager)

In my case, the plugin uses class "column"  and the result in the preview is (width is 50 %) :
![image](https://user-images.githubusercontent.com/5429548/132128220-480a0470-4cd0-41a8-9540-24003ab1451d.png)

This was solution:
`if(deftrue('USER_AREA') && (e_PAGE != "menus.php") )`
 but I think it is correct.
If it is, it should be added to e_header.php of _blank plugin

Otherwise:
- USER_AREA shouldn't be defined true in admin menu manager
- menu manager should use some custom classes, not common ones to avoid css conflicts

It is not so the big difference with bootstrap3 theme, but with theme, without bootstrap markup, it can look like this:
![image](https://user-images.githubusercontent.com/5429548/132128360-5fa6b8e0-ede7-42e6-bdc0-29c4d1856e5a.png)

THanks

**Comment by Jimmi08** [comment]
@Deltik 
According to your last fix, in e_header should be e_MENUMANAGER_ACTIVE != true used too?    {instead of (e_PAGE != "menus.php")) 
Big thanks

**Comment by Deltik** [comment]
I don't quite understand this issue.  Are you talking about `e107_plugins/_blank/e_header.php`?

If [`e_MENUMANAGER_ACTIVE === true`](https://github.com/e107inc/e107/blob/8fc922c126e36bc2a6e6b4c320d1b567090c85de/e107_admin/menus.php#L33-L33), then [`USER_AREA === true`](https://github.com/e107inc/e107/blob/8fc922c126e36bc2a6e6b4c320d1b567090c85de/e107_admin/menus.php#L33-L33), too.

**Comment by Deltik** [comment]
I don't quite understand this issue.  Are you talking about `e107_plugins/_blank/e_header.php`?

If [`e_MENUMANAGER_ACTIVE === true`](https://github.com/e107inc/e107/blob/8fc922c126e36bc2a6e6b4c320d1b567090c85de/e107_admin/menus.php#L33-L33), then [`USER_AREA === true`](https://github.com/e107inc/e107/blob/8fc922c126e36bc2a6e6b4c320d1b567090c85de/e107_admin/menus.php#L16), too.

**Comment by Jimmi08** [comment]
Yes, there is an example of how to use e_header correctly only on the front site, not in backend.  But it fails in the Menu manager (exactly theme preview and frontend css influences menu manager look - menus areas in my case). 
 
Ignore this, I thought that it could fix this too. Probably not.  I can live with e_PAGE. Thanks.

**Comment by Jimmi08** [comment]
It works correctly now, issue was in theme shortcodes and parsing magic shortcodes
